src/IDispatcher.cs

```diff
@@ -25,6 +26,6 @@ namespace Apache.NMS.ActiveMQ
     /// </summary>
     public interface IDispatcher
     {
-        void Dispatch(MessageDispatch messageDispatch);
+        Task Dispatch_Async(MessageDispatch messageDispatch);
```

Comment:
Why not `DispatchAsync`?

Reply:
there was collision with Session.DispatchAsync